### DISCUSSION TITLE
v6.0 Release ETA?

### DISCUSSION THREAD

**Discussion by cmitchell** [ravendb/ravendb#15759] [General]
Hi All,

  I was wondering if there's a set date, or even an estimated date for the v6.0 release? Also, are there nightly or RC builds available to play around with? (I didn't see any available under tags, but I figured it can't hurt to ask)

**Comment by ayende** [comment]
Here are the nightlies:
https://hibernatingrhinos.com/builds?product=RavenDB&version=6.0&channel=Nightly&pageNr=1

We're probably couple of months out from a more polished version, where we'll actively solicit community feedback
